packages/anoncreds/src/utils/getRevocationRegistries.ts

```diff
@@ -88,8 +88,10 @@ export async function getRevocationRegistriesForRequest(
             )
           }
 
-          const { tailsLocation, tailsHash } = revocationRegistryDefinition.value
-          const { tailsFilePath } = await downloadTailsFile(agentContext, tailsLocation, tailsHash)
+          const tailsFileService = agentContext.dependencyManager.resolve(AnonCredsModuleConfig).tailsFileService
+          const { tailsFilePath } = await tailsFileService.downloadTailsFile(agentContext, {
+            revocationRegistryDefinition,
```

Comment:
Is it clear that this will check if it's already downloaded? Or do you think we should do a check if it exists first?

Reply:
I renamed this to `getTailsFile`, so it will retrieve from the public URL or use a cached version if the TailsFileManager implementation does so.